pages.it/common/calibredb.md

```diff
@@ -18,8 +18,8 @@
 
 - Aggiungi uno o più ebook alla libreria:
 
-`calibredb add {{file1 file2 …}}`
+`calibredb add {{file1}} {{file2}}`
```

Comment:
This can be an arbitrary number of items.
```suggestion
`calibredb add {{file1}} {{file2}} {{fileN}}`
```

Reply:
Would it be possible to replace `path/to/file` with `file` in general? It looks much cleaner and readable and in most cases is redundent. In cases where it's important we could keep it.

Reply:
I'm a fan of `path/to/file` because it clearly indicates that it's a file _path_, as opposed to a file _name_ (because there are commands that take a file name, not path, e.g. [this](https://github.com/tldr-pages/tldr/pull/5767#discussion_r614225793)).  It also makes it easy to indicate if the path must be an absolute one, which, in fact, many tools require.

Reply:
`path/to/file` is our convention elsewhere, so generally speaking we prefer it over `file`. But `file` can be useful if you have many items, sure as here.